### PR TITLE
Moved all product flow model definitions to shared Products Repo

### PR DESCRIPTION
**Pull request by bothaellen**
Moved all Product Flow Model definitions to this repo

### CODE REVIEW

File: src/LightstonePlatform.Products/LightstonePlatform.Products.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (jaq316) [inline src/LightstonePlatform.Products/LightstonePlatform.Products.csproj]: Why this bump?

File: src/LightstonePlatform.Products/Models/ProductFlowHistoryModel.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿using LightstonePlatform.Products.Enums;
+using Newtonsoft.Json;
+using System;
+
+namespace LightstonePlatform.Products.Models
+{
+    public class ProductFlowHistoryModel
```
Comment (jaq316) [inline src/LightstonePlatform.Products/Models/ProductFlowHistoryModel.cs]: Should the `Model` not be dropped to align to the other changes in this PR?

It is also already in a `Models` namespace

**Review by jaq316 — CHANGES_REQUESTED** [review]

File: src/LightstonePlatform.Products/LightstonePlatform.Products.csproj (hunk 1 above)
Comment (bothaellen) [reply src/LightstonePlatform.Products/LightstonePlatform.Products.csproj]: reverted as this was a change needed when referencing the lightstone assemblies which was removed

File: src/LightstonePlatform.Products/Models/ProductFlowHistoryModel.cs (hunk 2 above)
Comment (bothaellen) [reply src/LightstonePlatform.Products/Models/ProductFlowHistoryModel.cs]: Have removed the Model postfix as per the comment

**Review by jaq316 — APPROVED** [review]

**Pull request merged**